sdk/monitor/Azure.Monitor.Query/src/Models/LogsBatchQueryResultStatus.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Monitor.Query.Models
+{
+    /// <summary>
+    /// Represents a status of a batch query response.
+    /// </summary>
+    public enum LogsBatchQueryResultStatus
+    {
+        /// <summary>
+        /// The query succeeded.
+        /// </summary>
+        Success,
+        /// <summary>
+        /// The query partially succeeded, both data and an error was returned.
```

Comment:
"at least one error"?

Reply:
This one is for an individual response in a batch. For log analytics, every request can partially fail.